cruise-control/src/yaml/responses/anomalyDetectorState.yaml

```diff
@@ -160,3 +170,6 @@ MeanTimeBetweenAnomaliesMs:
     DISK_FAILURE:
       type: number
       format: double
+    TOPIC_ANOMALY:
+      type: number
+      format: double
```

Comment:
-- We should consider prioritizing https://github.com/linkedin/cruise-control/issues/1041.
Sanity checking this file is non-trivial.

Reply:
Sure, I will prioritize the work.